src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough4.java

```diff
@@ -0,0 +1,118 @@
+/*
+FallThrough
+checkLastCaseGroup = true
+reliefPattern = (default)falls?[ -]?thr(u|ough)
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.fallthrough;
+
+public class InputFallThrough4 {
+    void methodFallThru(int i, int j, boolean cond) {
+        while (true) {
+            switch (i) {
+                case -1: // FALLTHRU
+                case 0:
+                case 26:
+                    switch (j) {
+                        case 1:
+                            continue;
+                        case 2:
+                            break;
+                        default:
+                        return;
+                    }
+                // fallthru
+            default:
+              // this is the last label
+                i++;
+            // fallthru
+         }
+      }
+   }
+
+   void methodFallThruCC(int i, int j, boolean cond) {
+      while (true) {
+          switch (i){
+          case 5:
+              i++;
+          // fallthru
+          }
+      }
+   }
+
+   void methodFall(int i, int j, boolean cond) {
+      while (true) {
+          switch (i){
+          case 5: // violation 'Fall through from the last branch of the switch statement'
+              i++;
```

Comment:
Please do `case 5: // violation 'Fall .* from the last branch of the switch statement'`
apply same for all others.

to be sure that `'Fall through` is not considered as relief comment for input file, regexp is too complicated to catch what form of wording is matching for Input. Lets ease reading of such Inputs for people.

Reply:
done